exporters/dsexporter/dsexporter.go

```diff
@@ -1,39 +1,115 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
+	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 )
 
+const service = "https://console-openshift-console.apps.stone-prd-rh01.pg1f.p1.openshiftapps.com/k8s/ns/appstudio-grafana/services"
+const check = "prometheus-appstudio-ds"
+var kubeconfig *string
+
 type CustomCollector struct {
-	requestCounter prometheus.Counter
+	konfluxProbeSuccess *prometheus.GaugeVec
 }
 
 // Creating a new instance of CustomCollector.
 func NewCustomCollector() *CustomCollector {
 	return &CustomCollector{
-		requestCounter: prometheus.NewCounter(prometheus.CounterOpts{
-			Name: "request_count",
-			Help: "Number of requests handled by the handler",
-		}),
+		konfluxProbeSuccess: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "konflux_probe_success",
```

Comment:
We dropped the use of the word `probe` as far us I understand. I think we should name it differently.